rbac/management/role/view.py

```diff
@@ -321,6 +324,19 @@ def update(self, request, *args, **kwargs):
         """
         return super().update(request=request, args=args, kwargs=kwargs)
 
+    @action(detail=True, methods=['get'])
+    def access(self, request, uuid=None):
+        """Return access objects for specified role."""
+        try:
+            role = Role.objects.get(uuid=uuid)
+        except (Role.DoesNotExist, ValidationError):
+            raise Http404
+
+        if role:
+            access = AccessSerializer(role.access, many=True).data
+            page = self.paginate_queryset(access)
+            return self.get_paginated_response(page)
```

Comment:
Maybe put these into the try block? 

Will there be a use case to make the default no pagination, like the one you did for /access endpoint? (currently default limit is 10)

Reply:
Hey @astrozzc! In this case, since the access objects aren't being used the same as the `/access/?application=` endpoint, we'll be keeping the default pagination setup. The experience team actually requested it all be paginated as well.

Reply:
Got it, thanks 👍

Reply:
How about putting these into the try block? That way we dont need the if statement

Reply:
@astrozzc Actually, we can probably just get rid of the conditional altogether now, since the `try` above will either return a role, or fail with a 404. Or is there another potential edge-case we should try to catch?

Reply:
> @astrozzc Actually, we can probably just get rid of the conditional altogether now, since the `try` above will either return a role, or fail with a 404. Or is there another potential edge-case we should try to catch?

Sure, both works!

Reply:
@astrozzc - updated!